src/LoadingSpinner/LoadingSpinner.tsx

```diff
@@ -58,35 +81,25 @@ export default class LoadingSpinner extends React.Component {
 
   logWarnings = () => {
     warn(() => {
-      const { parentNode } = ReactDOM.findDOMNode(this.refs.loadingSpinner);
-      const { position } = window.getComputedStyle(parentNode);
+      const { parentElement } = ReactDOM.findDOMNode(this.refs.loadingSpinner);
+      const { position } = window.getComputedStyle(parentElement!);
```

Comment:
this is just to make TS happy. There's no practical difference between `parentNode` and `parentElement`. Also I've used `parentElement!` because the only element whose `parentElement` is `null` is `document`.